site/en/tutorials/keras/save_and_load.ipynb

```diff
@@ -217,17 +217,17 @@
       "source": [
         "# Define a simple sequential model\n",
         "def create_model():\n",
-        "  model = tf.keras.models.Sequential([\n",
-        "    keras.layers.Dense(512, activation='relu', input_shape=(784,)),\n",
-        "    keras.layers.Dropout(0.2),\n",
-        "    keras.layers.Dense(10)\n",
-        "  ])\n",
+        "    model = tf.keras.models.Sequential([\n",
```

Comment:
Google tools all default to 2-space indent. This part of the change goes against that. LGTM otherwise.